definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -216,13 +216,6 @@
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "R"
   },
-  {
```

Comment:
@jacmurphy, as per our conversation, I'd have that in a separate PR.
This will need re-work after D&D goes live.

Reply:
Yep, well spotted. 
Here is said PR: https://github.com/hmcts/div-ccd-definitions/pull/638/files

I have removed it from the D&D file so that the tests pass and dont see it as a duplicate. 
Once D&D has been merged, this PR will be ready

Reply:
At the moment it's not that clear if DnD is going to be released tonight

Reply:
This is still here. I thought you were going to use a different PR.